tests/t000-ceed-f.f90

```diff
@@ -10,6 +10,7 @@ program test
 
       call ceedinit(trim(arg)//char(0),ceed,err)
       call ceeddestroy(ceed,err)
+      call ceeddestroy(ceed,err)
```

Comment:
I added this to test the double destroy case. I'm happy having it stay or go.

Reply:
Maybe just leave a comment, so that a reader in the future does not interpret this as a possible copy-paste mistake?

Reply:
Good idea. Updated